pages/posts/flight-of-a-byte.tsx

```diff
@@ -0,0 +1,112 @@
+                </p> 
+                <p>
+                 As mentioned, <code>client2</code> doesn&apos;t send the message “Hello” directly to the other group members. Instead, it puts the message in its own message log and updates the head. The other members learn about the new head which means there is a new message to fetch.
+                </p>
+                <h3>libp2p pubsub</h3>
+                <p>
+                 So now the question is, how does <code>client1</code> learn that <code>client2</code> says there is a new message log head? This uses libp2p pubsub where peers can subscribe to a topic and publish a pubsub message to that topic, or receive pubsub messages. (We say “pubsub message” to not confuse with the ordinary messages in the log which we&apos;ll discuss below.)
+                </p>
+                <p>
+                libp2p pubsub is a powerful tool with algorithms which allow it to scale to many peers and to make sure they all get the pubsub message. You can read more in its documentation, but for this blog post we only need to say that the Wesh group has a libp2p topic name like 
+                &nbsp;<code>
+                /orbitdb/bafyreihpdptqwgpuwu4ob6nusxebmdx6fm6mq5pma5hva6eu5u74pflo7e/2ac4e88c5272f900c76fd36989f9780e5b2c95d75d38fe5bcef0345b34bc4806_wesh_group_messages
+                    </code>
+                &nbsp;a pubsub message to this topic that contains the message log head. The libp2p on <code>client1</code> receives this pubsub message and sends it (code) internally to all processes which are subscribed to the topic. The Wesh application of <code>client1</code> has used ActivateGroup and is therefore subscribed to this topic and receives the pubsub message (code).
+                </p>
+                <h3>IPFS to fetch the message</h3>
+                <p>
+                libp2p pubsub doesn&apos;t care about the meaning of a pubsub message. It only makes sure that all subscribers get it. Remember that the next step is to synchronize the message log, which is handled by the local message store of <code>client1</code>. After checking that it&apos;s not a repeat, the pubsub message is forwarded (code) as an EventPubSubMessage to the message store which receives it (code). This confirms that the type of pubsub message is to update the message log head, and it checks to make sure the head was not already processed. (If <code>client1</code> has already received the same message log head, maybe from another peer, then it doesn&apos;t need to do more processing. This may seem trivial, but it&apos;s important because it&apos;s efficient to exchange these small “head” messages frequently between peers.)  
+                </p>
```

Comment:
Please include the code links for "is forwarded (code)" and "receives it (code)".